src/views/HomeView.vue

```diff
@@ -1,137 +1,115 @@
-              :size="buttonSize"
-            >
-              Play with AI
-            </v-btn>
-            <v-btn
-              variant="outlined"
-              class="mt-4"
-              href="https://discord.gg/9vrAZ8xGyh"
-              target="_blank"
-              :size="buttonSize"
-            >
-              Discord
-            </v-btn>
-            <h6 class="text-h6 mt-4">
-              Looking to Play?
-            </h6>
-            <p class="mt-0">
-              Our weekly play sessions are open to everyone!
-            </p>
-            <ul>
-              <li>Wednesday Nights at 8:30pm EST</li>
-              <li>Thursdays at 12pm EST</li>
-            </ul>
-            <p>
-              Can't find a match?
-              <v-btn variant="text" href="https://discord.gg/9vrAZ8xGyh">
-                Join our discord
+            <v-divider v-if="$vuetify.display.mdAndDown" color="surface-1" class="border-opacity-100" />
+            <v-window v-model="tab" class="pa-4 overflow-y-auto">
+              <v-window-item :value="TABS.PLAY">
+                <p v-if="playableGameList.length === 0" data-cy="text-if-no-game" class="text-surface-1">
+                  No Active Games
+                </p>
+                <div v-for="game in playableGameList" :key="game.id">
+                  <game-list-item
+                    :name="game.name"
+                    :p0ready="game.p0Ready ? 1 : 0"
+                    :p1ready="game.p1Ready ? 1 : 0"
+                    :game-id="game.id"
+                    :status="game.status"
+                    :num-players="game.numPlayers"
+                    :is-ranked="game.isRanked"
+                    @error="handleSubscribeError(game.id, $event)"
+                  />
+                </div>
+              </v-window-item>
+              <v-window-item :value="TABS.SPECTATE">
+                <p v-if="specateGameList.length === 0" data-cy="no-spectate-game-text" class="text-surface-1">
+                  No Games Available to Spectate
+                </p>
+                <div v-for="game in specateGameList" :key="game.id">
+                  <game-list-item
+                    :name="game.name"
+                    :p0ready="game.p0Ready ? 1 : 0"
+                    :p1ready="game.p1Ready ? 1 : 0"
+                    :game-id="game.id"
+                    :status="game.status"
+                    :num-players="game.numPlayers"
+                    :is-ranked="game.isRanked"
+                    :is-spectatable="true"
+                    :disable-spectate="game.isOver"
+                    @error="handleError"
+                  />
+                </div>
+              </v-window-item>
+            </v-window>
+          </div>
+        </v-col>
+      </v-row>
+      <v-row>
+        <v-col class="home-card-games" :cols="$vuetify.display.mdAndUp ? 8 : 12">
+          <div class="mx-auto my-2 homeContent">
+            <create-game-dialog @error="handleError" />
```

Comment:
Let's add more margin on either side of the create game dialog. It's a bit squished at the moment. I'm not positive we can put this directly on the create-game-dialog instance, but let's try and/or fiddle with it until the button has space above and below it
```suggestion
            <create-game-dialog class="my-4" @error="handleError" />
```